development/tests/test_views_forms.py

```diff
@@ -50,3 +50,23 @@ def test_get_queryset_my_bots(self):
         request.user = self.user1
         response = MyBotsView.as_view()(request)
         self.assertEqual(response.status_code, 200)
+
+    def test_get_form_add_bot_get(self):
+        request = self.factory.get('development:addbot')
+        request.user = self.user1
+        response = AddBotView.as_view()(request)
+        self.assertEqual(response.status_code, 200)
+
+    @patch('development.views.messages')
+    def test_get_form_add_bot_post_ok(self, patched_message):
+        request = self.factory.post('development:addbot', {'name': 'botty'})
+        request.user = self.user1
+        response = AddBotView.as_view()(request)
+        self.assertEqual(response.status_code, 302)
```

Comment:
After this assert you can check if now there is a new Bot in the database called `botty`

Reply:
done